packages/babel-cli/test/fixtures/babel/--copy-files --include-dotfiles with ignore/out-files/lib/foo/.foo.js

```diff
@@ -1 +0,0 @@
-a;
```

Comment:
🤔Why is `.foo.js` not copied?

Reply:
we've --include-dotfile to Include dotfiles when compiling and copying non-compilable files.

https://github.com/rajasekarm/babel/blob/master/packages/babel-cli/src/babel/options.js#L148

Reply:
Yeah, it _was_ copied before this PR. But now .foo.js has been removed from the out files, which means it is _not_ copied.

Reply:
I got it, I'll check.

Reply:
I've that file in my branch, https://github.com/rajasekarm/babel/blob/master/packages/babel-cli/test/fixtures/babel/--copy-files%20--include-dotfiles%20with%20ignore/out-files/lib/foo/.foo.js

I dono why github shows this as removed.

Reply:
The linked one is your master branch, not the PR branch :stuck_out_tongue:

Reply:
Yeah, I'm check this.

Reply:
Found the root cause of this bug, I'm fixing it.

Reply:
.foo.js is inside src/foo and it is in ignored list. So it is not copied.